### ISSUE TITLE
AttributeError: module 'requests.packages' has no attribute 'urllib3'

### ISSUE THREAD

**Issue by Odianosen25** [home-assistant/appdaemon#344]
I just reinstalled AppD on another system and on version 3.0.2, and I get the following errors:

```
2018-07-15 12:44:53.573705 WARNING AppDaemon: ------------------------------------------------------------
2018-07-15 12:44:53.586863 WARNING AppDaemon: ------------------------------------------------------------
2018-07-15 12:44:53.587876 WARNING AppDaemon: Unexpected error initializing app: home_auto_intent:
2018-07-15 12:44:53.588884 WARNING AppDaemon: ------------------------------------------------------------
2018-07-15 12:44:53.591961 WARNING AppDaemon: Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/appdaemon/appdaemon.py", line 2045, in check_app_updates
    self.init_object(app)
  File "/usr/local/lib/python3.5/dist-packages/appdaemon/appdaemon.py", line 1561, in init_object
    modname = __import__(app_args["module"])
  File "/home/pi/.appdaemon/conf/apps/jarvis_apps/home_auto_intent.py", line 1, in <module>
    import appdaemon.plugins.hass.hassapi as hass
  File "/usr/local/lib/python3.5/dist-packages/appdaemon/plugins/hass/hassapi.py", line 9, in <module>
    requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
AttributeError: module 'requests.packages' has no attribute 'urllib3'

2018-07-15 12:44:53.593012 WARNING AppDaemon: ------------------------------------------------------------
```
For all the apps that need some form of service call to hass. Strange this is I had done same like 2 weeks ago and I didn't have this issues.

I looked up the internet and I found out there seems to be something about requests and urllib3.

I have the following versions

Requests 2.19.1
Urllib3 1.23

I changed the following in the hassapi.py file and it seems to fixed it. But I can't be certain since I don't use https without certificate.

from 
```
from requests.packages.urllib3.exceptions import InsecureRequestWarning
requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
```
to

```
try:
    from requests.packages.urllib3.exceptions import InsecureRequestWarning
    requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
except:
    import urllib3
    from urllib3.exceptions import InsecureRequestWarning
    urllib3.disable_warnings(InsecureRequestWarning)
```

Any idea pls?

**Comment by ReneTode** [comment]
first i must say that its great that you have 3.02.
but where did you get it, because 3.01 is the latest release when i am not mistaken.

i think some libs are updated and generate trouble.
first it was pyyaml, now the requests lib.

i guess that requests has removed or renamed the urllib3 package.
i think your solution is the best for now untill we find out what has changed.
to change it in AD we need andrew and he is still indisposed for the moment so we need to wait for that.

lets leave this issue standing so he knows he has to look at it.

**Comment by Odianosen25** [comment]
Hello @ReneTode,

Thanks for the response, and to be honest I have no clue where that came from. Even the previous system I installed AppD on is saying 3.0.2 and I am aware the latest should be 3.0.1.

Yes a lot of packages, as I am migrating my code to a new more reliable hardware and I have been struggling with packages changes. The worst is still pendulum :(.

Anyways as you said, I will wait for the update from @acockburn.

Thanks and regards

**Comment by Odianosen25** [comment]
Yep just `sudo pip3 install appdaemon`.

But I am not running it in a virtual environment, as both instances I have AppD as a standard app in the system. Not sure if that makes any difference. I then have Hass in a virtual environment.

Not sure if that means anything.

**Comment by ReneTode** [comment]
shouldnt make a difference.
i just recently istalled a second instance from AD in a second venv.
it just says AD 3.0.1 in the logs when i startup.
that was less then 2 weeks ago without any errors.

**Comment by Odianosen25** [comment]
Same thing, as I had one installed like 3/4 weeks ago and it was all fine.

Not sure why this popped up, and its very strange really.

But anyways so far things working fine. Just as the pyyaml gave me a kick, well this just showing its head.

**Comment by Odianosen25** [comment]
@ReneTode,

I think I kind of figured out how I got the `3.0.2`. When trying to install AppD on my new system, I didn't just try `pip3 install` first. I had tried installing directly from Github, and I wasn't sure if it had correctly installed, so I then did the `pip3 install`.

I think it installed properly the first time, and I didn't realise it. As part of the confusion I was having, was if to install it system wide. Maybe it is this version of the code, that now caused this problem to exist.

Regards
